### PR TITLE
azurerm_virtual_network - add vnet_peering_address_list as datasource output for vnet

### PR DESCRIPTION
**Pull request by danielino**
i try to get address_spaces for vnet peerings but it's not very simple.

an example of what i try to do:

```
/*
locals {
  vnet_in_peerings = data.azurerm_virtual_network.vnet.vnet_peerings
  vnet_lists = [for vnet_name, vnet_id in local.vnet_in_peerings: [vnet_name, regex(".+resourceGroups\\/(\\w.+)\\/providers", vnet_id)[0]]]
  vnet_peering_address_space_lists = []
}

# this code doesn't run, because terraform are not able to create a plan because local.vnet_lists is populated with an apply
data "azurerm_virtual_network" "vnets-peering" {
  #for_each = local.vnet_lists
  count = length(local.vnet_lists)
  #name = each.key
  name = local.vnet_lists[count.index][0]
  resource_group_name = local.vnet_lists[count.index][1]
  #resource_group_name = each.value
}

locals {
  vnet_peering_address_space_lists = [for vnet_name, vnet in data.azurerm_virtual_network.vnets-peering: vnet.address_spaces]
}

```

so i modified the azurerm_virtual_network datasource having added an attribute `vnet_peerings_address_lists` that is a list of address spaces of the peered vnets

### CODE REVIEW

**Comment by danielino** [comment]
hei, there is a timeout in linting action:

```
 level=error msg="Timeout exceeded: try increasing it by passing --timeout option"
  
  Error: golangci-lint exit with code 4
  Ran golangci-lint in 4639044ms
```

this not depends on the code i've writed

**Comment by testotxt** [comment]
hi @catriona-m  any news on this pull request?

thanks

File: internal/services/network/virtual_network_data_source.go [hunk 1]
```diff
@@ -69,6 +69,13 @@ func dataSourceVirtualNetwork() *pluginsdk.Resource {
 					Type: pluginsdk.TypeString,
 				},
 			},
+			"vnet_peerings_address_lists": {
```
Comment (katbyte) [inline internal/services/network/virtual_network_data_source.go:72]: we don't include "list" in the property name, so this should be
```suggestion
			"vnet_peerings_addresses": {
```

and

**Review by katbyte — CHANGES_REQUESTED** [review]
Thanks @danielino - in addition to my comment inline we'll need to update the documentation to reflect this new property

**Comment by danielino** [comment]
> Thanks @danielino - in addition to my comment inline we'll need to update the documentation to reflect this new property

done, i've updated the documentation and attribute name

**Pull request merged**